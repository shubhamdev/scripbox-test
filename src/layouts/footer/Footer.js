import React from "react";

function Footer(props) {
  return <div className="footer">footer</div>;
}

export default Footer;
