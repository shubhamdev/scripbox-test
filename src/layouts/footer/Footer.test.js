import React from "react";
import { unmountComponentAtNode } from "react-dom";
import { shallow } from "enzyme";
import Footer from "./Footer";

let container = null;

beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe("Footer", () => {
  it('should render Footer correctly in "debug" mode', () => {
    const component = shallow(<Footer debug />);
    expect(component).toMatchSnapshot();
  });
  it("should render Footer correctly with no props", () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
