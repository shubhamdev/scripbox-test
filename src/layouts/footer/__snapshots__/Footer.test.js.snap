// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer correctly in "debug" mode 1`] = `
<div
  className="footer"
>
  footer
</div>
`;

exports[`Footer should render Footer correctly with no props 1`] = `
<div
  className="footer"
>
  footer
</div>
`;
